tests/register/register.go

```diff
@@ -120,6 +146,8 @@ func (test Test) RunCoreOSInstall(t *testing.T, loopDevice string, opts ...strin
 		opts = append(opts, "-c", cloudinitPath)
 	}
 
+	t.Logf("Running coreos-install via the following command: %s %s", test.BinaryPath, strings.Join(opts, " "))
```

Comment:
Please trim this message down.  The command line is the important part, and the extra words will make it harder to skim the logs.